docs/api.rst

```diff
@@ -1499,6 +1499,10 @@ of :class:`enum.Enum`.
 
         The user is a Known Spammer.
 
```

Comment:
For consistency, no newline before the attribute

Reply:
1b088b1139377eefa202d03493666f7d43575af8